src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
```

Comment:
I don't think a lifecycle method will work here because we can't rely on this component having been mounted before registering the component. I'm not sure what the best way to handle this, what do you think?